dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -304,22 +357,82 @@ def parameters(self):
             return parameters
 
     def sample_dqm(self, dqm, time_limit=None, compressed=False, **kwargs):
-        """Sample from the specified binary quadratic model.
+        """Sample from the specified discrete quadratic model.
 
         Args:
-            bqm (:obj:`dimod.DiscreteQuadraticModel`):
-                The binary quadratic model.
+            dqm (:obj:`dimod.DiscreteQuadraticModel`):
+                Discrete quadratic model (DQM).
+
+            time_limit (int, optional):
+                Maximum run time, in seconds, to allow the solver to work on the
+                problem. Must be at least the minimum required for the number of
+                problem variables, which is calculated and set by default.
+                The minimum time for a hybrid solver is specified as a
+                piecewise-linear curve defined by a set of floating-point pairs,
+                the `minimum_time_limit` field under
+                :attr:`~dwave.system.samplers.LeapHybridBQMSampler.properties`.
+                The first element in each pair is a combination of the numbers of
+                interactions, variables, and cases that reflects the "density" of
+                connectivity between the problem's variables;
+                the second is the minimum required time. The minimum time for any
+                particular problem size is a linear interpolation calculated on
+                two pairs that represent the relevant range for the given problem.
+                For example, if `LeapHybridSampler().properties["minimum_time_limit"]`
+                returns `[[1, 0.1], [100, 10.0], [1000, 20.0]]`, then the minimum
+                time for a problem of "density" 50 is 5 seconds, the linear
+                interpolation of the first two pairs that represent problems with
+                "density" between 1 to 100.
+                :meth:`~dwave.system.samplers.LeapHybridBQMSampler.min_time_limit`
+                calculates the minimum time for your problem.
 
-            time_limit (int):
-                The maximum run time in seconds.
+            compressed (binary, optional):
+                Compresses the DQM data when set to True.
 
             **kwargs:
                 Optional keyword arguments for the solver, specified in
-                :attr:`~dwave.system.samplers.LeapHybridSampler.parameters`.
+                :attr:`~dwave.system.samplers.LeapHybridBQMSampler.parameters`.
 
         Returns:
             :class:`dimod.SampleSet`: A sample set.
 
+    Examples:
```

Comment:
why not just refer back to the class description for a second example?

Reply:
Done. Seems a shame, though, we have so few other examples that involve pizza and wine.